data/experiment-recipe-samples/rollout-desktop.json

```diff
@@ -0,0 +1,42 @@
+{
+  "slug": "rollout-desktop-example",
+  "appId": "firefox-desktop",
+  "appName": "firefox_desktop",
+  "channel": "",
+  "endDate": null,
+  "branches": [
+    {
+      "slug": "control",
+      "ratio": 1,
+      "feature": {
+        "value": {},
+        "enabled": true,
+        "featureId": "no-feature-firefox-desktop"
+      }
+    }
+  ],
+  "outcomes": [],
+  "arguments": {},
+  "probeSets": [],
+  "startDate": null,
+  "targeting": "version|versionCompare('95.!') >= 0 && 'app.shield.optoutstudies.enabled'|preferenceValue",
+  "featureIds": ["no-feature-firefox-desktop"],
+  "application": "firefox-desktop",
+  "bucketConfig": {
+    "count": 5000,
+    "start": 0,
+    "total": 10000,
+    "namespace": "firefox-desktop-no-feature-firefox-desktop-rollout-1",
+    "randomizationUnit": "normandy_id"
+  },
+  "schemaVersion": "1.5.0",
+  "userFacingName": "Testing Rollout Desktop",
+  "referenceBranch": "control",
+  "proposedDuration": 28,
+  "isRollout": true,
```

Comment:
oh, I have this implemented as `is_rollout` in my current Desktop patch (not yet landed). This is what we're going with `isRollout` right?

Reply:
I'd say yeah, this is what we're going with - if only to follow the camelCase convention of the other properties